Client/Client.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+
+namespace RedGear 
+{
+    namespace RedGearClient 
+    {
+        public class Client 
+        {
+            public static VerifySystem.Verify Launch() 
+            {
+                try 
+                {
+                    NetworkSystem.PackageManager.StartConnexion();
+                    return new VerifySystem.SuccessSystem.OkSuccess("hehe", "Content");
+                } 
+                catch (Exception e) 
+                {
+                    Console.WriteLine(e.Message);
+                    return new VerifySystem.ErrorSystem.BadRequestError("jsp", e.Message);
```

Comment:
```suggestion
                    return new VerifySystem.ErrorSystem.BadRequestError("Bad request error / Something bad happened", e.Message);
```
Pareil, à modifier pour plus de lisiblité